File: src/EducationSystem/EducationSystem.WebApp/App/src/models/Theme.ts
import Model from './Model'

export default class Theme extends Model {
  public Name: string = ''
  public DisciplineId?: number
  public Questions?: Array<object>
  public Order?: number

  constructor(disciplineId?: number) {
    super()

    this.DisciplineId = Number(disciplineId)
  }
}

File: src/EducationSystem/EducationSystem.Models/Rest/Theme.cs
using System.Collections.Generic;
using EducationSystem.Models.Rest.Basics;

namespace EducationSystem.Models.Rest
{
    public class Theme : Model
    {
        public string Name { get; set; }

        public int DisciplineId { get; set; }

        public int? Order { get; set; }

        public List<Question> Questions { get; set; }
    }
}